admin-base/materialize/custom/_navbar.scss

```diff
@@ -139,6 +139,53 @@ nav {
             background-color: var(--pcms-orange-hover);
           }
         }
+
+        .user-info {
+          display: flex;
+          flex-direction: column;
+          width: 100%;
+          line-height: 2rem;
+
+          .row {
+            margin: .2rem 1rem .2rem 0;
+            padding-left: 1rem;
+
+            &:first-child {
+              margin-top: 0;
+            }
+
+            &:last-child {
+              margin-bottom: 0;
+            }
+
+            > p {
+              margin: 0 0 0 -5px;
+              font-weight: bold;
+            }
+          }
+
+          .copy-username {
+            float: right;
+            color: #333;
+            font-size: 13px;
```

Comment:
Maybe we use `em` or `rem` instead of `px`?

Reply:
since this is relative to the given font-size: 15px of the text we should keep px here too